ui/mirage/common.js

```diff
@@ -39,6 +41,7 @@ export function generateNetworks(options = {}) {
       CIDR: '',
       IP: faker.internet.ip(),
       MBits: 10,
+      Mode: faker.random.arrayElement(NETWORK_MODES),
```

Comment:
TIL: `arrayElement`. I wrote my own `pickOne` helper for this. Add getting rid of `pickOne` to the tech debt list 🙈